ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -103,10 +103,10 @@
       {{ item.registeringParty || item.submittingParty || '' }}
     </td>
     <td
-      v-if="inSelectedHeaders('ownerNames')"
+      v-if="inSelectedHeaders('ownerNames') && isPpr"
       :class="isChild || item.expanded ? $style['border-left']: ''"
     >
-      {{ item.ownerNames }}
+      {{ item.ownerNames && isPpr}}
```

Comment:
I'm not sure i understand the `isPpr` in the template. We've got a computed value in `item.ownerNames` which i'm assuming is a string or something followed by a boolean in `isPpr`. 
Was the intention to short circuit `item.ownerNames`?
If so, there is no fallback value either

Reply:
Good catch... Not necessary,